src/VodafoneChannel.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace NotificationChannels\Vodafone;
+
+use Illuminate\Notifications\Notification;
+use NotificationChannels\Vodafone\Exceptions\CouldNotSendNotification;
+
+class VodafoneChannel
+{
+    /**
+     * @var string Vodafone's API endpoint
+     */
+    protected $endpoint = 'https://www.smsertech.com/apisend';
```

Comment:
Do you have a copy or link to the API docs?

Reply:
Yes I do, Vodafone only provides a PDF of the documentation, which I have now included in the package files.

Reply:
Thanks :)